### DISCUSSION TITLE
Newbie question: Module not found: Error: Package path . is not exported from package ....exports field in package.json

### DISCUSSION THREAD

**Discussion by KingsGithub** [ghiscoding/aurelia-slickgrid#1285] [Q&A] [answered]
I'm trying to create the most basic grid and have installed aurelia and aurelia-slickgrid.
After including the registration in main.ts with aurelia.use.plugin(PLATFORM.moduleName('aurelia-slickgrid'));
I get the above error which points to this part in the package.json file in the dist folder:

 "exports": {
    ".": {
      "types": "./dist/index.d.ts",
      "import": "./dist/index.js"   
    },
    **"./package.json": "./package.json"**
  },
Can anyone give me some guidance as to what the real cause is and what to do to fix it?
Thanks.

**Comment by ghiscoding** [comment, marked as answer]
You seem to be using Aurelia 1 and for that you will need to make sure to use Aurelia-Slickgrid not higher than 6.x because all newer versions are for Aurelia 2 (versions compatibility are mentioned on the main readme page). Then you should read the Wiki [How to Step by Step](https://github.com/ghiscoding/aurelia-slickgrid/wiki/HOWTO--Step-by-Step)

**Reply by KingsGithub** [reply]
Thanks for the response. Much appreciated.
I have the following versions:
Aurelia:
"name": "aurelia",
  "version": "2.0.0-beta.22",
 Aurelia CLI:
  "name": "aurelia-cli",
  "version": "3.0.3",
  
 Just for perspective - I was working on a project when trying out slickgrid I ran into this issue. So I decided to start off with a clean slate and created a completely new project with nothing in it - straight off the command "au new" and ran it using "npm start" - it worked as expected. Then I installed slickgrid using "npm --save install aurelia-slickgrid bootstrap", registered it in main.ts and it immediately gave the error.

**Reply by ghiscoding** [reply]
I said that because you seem to use Au 1 because of the use of `PLATFORM.moduleName()` which is no longer needed and shouldn't be used at all in Au 2. If you're on Au 2 then you should use the latest [Documentation](https://ghiscoding.gitbook.io/aurelia-slickgrid/getting-started/quick-start) available on GitBook. Personally I haven't tried Au 2 with Vite yet, all the [Aurelia-Slickgrid-Demos](https://github.com/ghiscoding/aurelia-slickgrid-demos/tree/master/webpack-bs5-demo) are using Au 2 + WebPack

**Comment by KingsGithub** [comment]
Mmmmhhhh, I see that when I use au new it suggests I **"use Aurelia 1 with xyz options".** Looks like that might be the problem. I'll see about getting that changed.

**Reply by KingsGithub** [reply]
OK, finally got it to at least run without an error:
In a new project :
I used **"npx makes aurelia/new"** 
to choose option **Aurelia 2 with Typescript and Vite**
then ran **npm install --save aurelia-slickgrid**
then ran **npm install --save dompurify**
then in the main.ts -- copying from the DEMO Examples code
import DOMPurify from 'dompurify';
import  { AureliaSlickGridConfiguration } from 'aurelia-slickgrid'
Aurelia
.register(AureliaSlickGridConfiguration.customize(config => {
    // change any of the default global options
    config.options.gridMenu!.iconCssClass = 'mdi mdi-menu';
    // we strongly suggest you add DOMPurify as a sanitizer
    config.options.sanitizer = (dirtyHtml) => DOMPurify.sanitize(dirtyHtml, { ADD_ATTR: ['level'], RETURN_TRUSTED_TYPE: true });
  }))
   .app(MyApp)
  .start();

**Thanks for the heads-up pointer to Aurelia 1**
